scala/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.1.6
```

Comment:
I needed to bump this to 1.3.3, for some reason the hosted version of 1.1.6 that my `sbt` tried to download failed to be retrieved.

It could be a problem with the mirror, not sure.